transformer_engine/jax/cpp_extensions/attention.py

```diff
@@ -17,17 +17,18 @@
 from jax.sharding import PartitionSpec, NamedSharding
 from jax import ffi
 
-from transformer_engine.jax.attention import CPStrategy, SequenceDescriptor
-
-from transformer_engine import transformer_engine_jax
-from transformer_engine.transformer_engine_jax import (
+import transformer_engine_jax
+from transformer_engine_jax import (
```

Comment:
This is kind of unrelated to this PR but why do you need both of these lines?

Reply:
There are other functions from `transformer_engine_jax` which are used directly instead of importing it separately in the next line, such as `transformer_engine_jax.get_fused_attn_backend(...)`.  Not sure the rationale behind doing both styles but we could unify it in a separate (pure refactor) PR.